REST Bindings/openapi.yaml

```diff
@@ -958,56 +800,80 @@ paths:
+            GS1-EPC-Format:
+              $ref: '#/components/headers/GS1-EPC-Format'
+            GS1-CBV-XML-Format:
+              $ref: '#/components/headers/GS1-CBV-XML-Format'
+            GS1-EPCIS-Extensions:
+              $ref: '#/components/headers/GS1-EPCIS-Extensions'
+            GS1-CBV-Extensions:
+              $ref: '#/components/headers/GS1-CBV-Extensions'
+            Link:
+              example: '<https://example.com/epcs?perPage=30&nextPageToken=3A1550673874978%2Ci%3AU6D7DENAKwM2gQRRwGrataeq>; rel="next"'
+              schema:
+                $ref: '#/components/schemas/Link'
+          description: |
+            Returns a list of electronic product codes. If there are more electronic product codes than specified by the
+            `perPage` parameter, the client will be given the URL to retrieve more electronic product codes in the
+            `Link` header.
+            If the client specifies extension mappings, the response will use them where they match. Otherwise,
+            the full resource address is used.
+          content:
+            application/json:
+              schema:
+                example:
+                  - urn:epc:id:sgtin:0614141.107346.2018
+                  - https://example.com/01/04012345123456
+                  - urn:epc:id:sgtin:0614141.107346.2017
+                type: array
+                items:
+                  type: string
+                  format: uri
         '401':
           $ref: '#/components/responses/401UnauthorizedRequest'
         '403':
           $ref: '#/components/responses/403ClientUnauthorized'
-        '404':
-          $ref: '#/components/responses/404ResourceNotFound'
-        '414':
-          $ref: '#/components/responses/414URLTooLong'
         '406':
           $ref: '#/components/responses/406NotAcceptable'
         '500':
           $ref: '#/components/responses/500ImplementationException'
 
-  /epcs/{epc}/events/{eventID}:
+  /epcs/{epc}:
     parameters:
       - $ref: '#/components/parameters/EPC'
-      - $ref: '#/components/parameters/EventID'
     options:
       tags:
         - 'Discovery'
-      summary: Query metadata for the endpoint to access an individual EPCIS event.
+      summary: Query the metadata of the endpoint to access EPCIS events by EPC in a URN or constrained GS1 Digital Link form (URL encoded).
       description: |
         EPCIS 2.0 supports a number of custom headers to describe custom vocabularies and support multiple versions
         of EPCIS and CBV. The `OPTIONS` method allows the client to discover which vocabularies and EPCIS and CBV
         versions are used.
       responses:
         '204':
-          $ref: '#/components/responses/204TopLevelResourcesDiscovery'
+          $ref: '#/components/responses/204EventsEndpointDiscovery'
     get:
       tags:
         - 'Events'
-      summary: Returns an individual EPCIS event.
+      summary: Returns EPCIS events with the electronic product code.
```

Comment:
Should we illustrate against which attributes of events the epc (electronic product code) will be matched?